test/stylelintrc-css-modules/main.css

```diff
@@ -0,0 +1,49 @@
+/*
+Sample CSS showing syntax extensions for CSS Modules with variables.
+https://github.com/css-modules/css-modules
+*/
+
+$white: #fff;
+$gray-light: #d4d4d6;
+$gray-middle: #7e8484;
+$primary-light: #00909a;
+$alert-error: #ce4657;
+$alert-success: #a6d887;
+$size-h6: 1.6rem;
+
+.field {
```

Comment:
`composes` is the prime example of something unique to CSS modules.  This is one of the reasons I like it so much - it's really really close to vanilla CSS.

Reply:
Nice. I'm starting to use css-modules on some projects. It's great